docs/content/preview/yugabyte-voyager/migrate/migrate-steps.md

```diff
@@ -152,23 +210,34 @@ The `yb-voyager export schema` command extracts the schema from the source datab
 {{< note title="Renaming index names for MySQL" >}}
 
 YugabyteDB Voyager renames the indexes for MySQL migrations while exporting the schema.
+
 MySQL supports two or more indexes to have the same name in the same database, provided they are for different tables. Similarly to PostgreSQL, YugabyteDB does not support duplicate index names in the same schema. To avoid index name conflicts during export schema, yb-voyager prefixes each index name with the associated table name.
 
 {{< /note >}}
 
-{{< note title="Usage for source_db_schema" >}}
+You specify the schema(s) to migrate from the source database using the `db-schema` parameter (configuration file), or `--source-db-schema` flag (CLI).
 
-The `source_db_schema` argument specifies the schema of the source database.
+- For MySQL, currently this argument is not applicable.
+- For PostgreSQL, this argument can take one or more schema names separated by comma.
+- For Oracle, this argument can take only one schema name and you can migrate _only one_ schema at a time.
 
-- For MySQL, currently the `source-db-schema` argument is not applicable.
-- For PostgreSQL, `source-db-schema` can take one or more schema names separated by comma.
-- For Oracle, `source-db-schema` can take only one schema name and you can migrate _only one_ schema at a time.
+Run the command as follows:
 
-{{< /note >}}
+{{< tabpane text=true >}}
 
-An example invocation of the command with required arguments is as follows:
+  {{% tab header="Config File" lang="config" %}}
 
 ```sh
+yb-voyager export schema --config-file <path-to-config-file>
+```
+
+You can specify additional `export schema` parameters in the `export-schema` section of the configuration file. For more details, refer to the `offline-migration.yaml` template.
```

Comment:
You have to link to the "offline-migration.yaml" template everywhere I believe.

Reply:
to github link right?